.github/workflows/docs.yml

```diff
@@ -33,10 +33,10 @@ jobs:
 
       - name: Install dependencies
         run: |
-          uv pip install .[test,docs] --system
+          uv pip install ".[test,docs]" --system
 
       - name: Install extras for tutorial generation
-        run: uv pip install ".[graphpes,mace]" --system
+        run: uv pip install ".[graphpes,mace,metatensor]" --system
```

Comment:
Note, this may need to be refactored in future due to incompatible optional dependencies between tutorials?

Reply:
Yeah I am tracking this, will fix it if it becomes a problem